runtime/tests/tests.rs

```diff
@@ -261,17 +387,21 @@ fn staking_validators_should_receive_equal_transaction_fee_reward() {
 				balance_amount + transfer_amount
 			);
 
-			// Check total_issurance is adjusted
-			assert_eq!(
-				GenericAsset::total_issuance(&CENTRAPAY_ASSET_ID),
-				previous_total_issuance - remainder
-			);
+			// Check if stash account balances are not yet changed
+			for validator in validators.clone() {
+				let (stash, _) = validator;
+				assert_eq!(
+					<GenericAsset as MultiCurrencyAccounting>::free_balance(&stash, Some(CENTRAPAY_ASSET_ID)),
+					balance_amount
+				);
+			}
 
+			start_era(2);
 			for validator in validators {
 				// Check tx fee reward went to the stash account of validator
 				assert_eq!(
 					<GenericAsset as MultiCurrencyAccounting>::free_balance(&validator.0, Some(CENTRAPAY_ASSET_ID)),
-					balance_amount + fee_reward
+					balance_amount + per_fee_reward + per_staking_reward
```

Comment:
Can we use the same pattern: `let (stash, _) = validator;` as above to keep the checks consistent?